release notes/v0.44.0.md

```diff
@@ -0,0 +1,221 @@
+k6 v0.44.0 is here! 🎉 This release includes:
+
+- A new experimental `webcrypto` module implementing (partially) the [Web Crypto API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API) specification.
+- A sampling option for the experimental tracing module.
+- Memory usage improvements.
+- Bug fixes and UX improvements.
+
+Some highlights from the browser module are:
+
+- `locator.click` is now asynchronous, which is a breaking change.
+- `browserContext.addCookies` has now been implemented.
+- `browserType.Connect` has been implemented so k6 can now connect to an already running Chrome/Chromium browser instance.
+- Web vitals are natively supported when working with the browser module.
+
+## Breaking changes
+
```

Comment:
I suggest adding the following sentence at the beginning of this section.


```suggestion
The browser module is still in an `experimental` stage, and therefore breaking changes are expected as we are improving the APIs to make them more user-friendly.
```

This module has many breaking changes, which may look "bad" if the reader doesn't know this is expected.